tools/run_tests/generate_reports.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2015, Google Inc.
```

Comment:
Nit: I'd be leaning towards some other module name than "generate_reports" which looks more like a function name. 
Just "report" sounds good (if it doesn't collide with some standard python module name). "report_generator" or "report_utils" also sound good.